<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>985</width>
    <height>588</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>985</width>
    <height>588</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>985</width>
    <height>588</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Youtube Downloading Software</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="edit_link">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>901</width>
      <height>41</height>
     </rect>
    </property>
    <property name="inputMask">
     <string/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="placeholderText">
     <string>Youtube video or playlist link</string>
    </property>
   </widget>
   <widget class="QListWidget" name="listWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>240</y>
      <width>961</width>
      <height>331</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="button_download">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>200</y>
      <width>171</width>
      <height>33</height>
     </rect>
    </property>
    <property name="text">
     <string>Download</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Use proxies</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>100</y>
      <width>181</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>(slower but safer download)</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>170</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <item>
     <property name="text">
      <string>*.mp3</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>*.mp4</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>140</y>
      <width>251</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Choose format to download</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>570</y>
      <width>291</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="text">
     <string>(c) 2018 Bogdan Caleta Ivkovic All Rights Reserved</string>
    </property>
    <property name="textFormat">
     <enum>Qt::AutoText</enum>
    </property>
    <property name="alignment">
     <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
    </property>
   </widget>
   <widget class="QToolButton" name="button_destination">
    <property name="geometry">
     <rect>
      <x>920</x>
      <y>20</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
